src/js/serial_backend.js

```diff
@@ -338,10 +340,6 @@ function processBoardInfo() {
             MSP.send_message(MSPCodes.MSP_RESET_CONF, buffer, false);
 
             dialog.close();
-
-            GUI.timeout_add('connecting', function () {
```

Comment:
Why are you removing this?

Reply:
ups i have removed validation timeout,fixed now. Also readded old timeout killer,I thought that having the other one, we wouldn't need it.